openc3/lib/openc3/script/screen.rb

```diff
@@ -25,7 +25,7 @@ def get_screen_list(scope: $openc3_scope)
         endpoint = "/openc3-api/screens"
         # Pass the name of the ENV variable name where we pull the actual bucket name
         response = $api_server.request('get', endpoint, scope: scope)
-        if response.nil? || response.code != 200
+        if response.nil? || response.status != 200
```

Comment:
This is breaking to other users (including my current CFDP code).  Can we make code/status aliases (both work?)